zuul-core/src/main/java/com/netflix/netty/common/throttle/RejectionUtils.java

```diff
@@ -220,6 +235,7 @@ private static FullHttpResponse createRejectionResponse(
         if (closeConnection) {
             headers.set(HttpHeaderNames.CONNECTION, "close");
         }
+        rejectionHeaders.forEach(headers::add);
```

Comment:
headers.addAll(rejectionHeaders.forEach); 

?

Reply:
I don't see an `addAll` method on that object.